src/AggregateFunctions/QuantileBFloat16Histogram.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <AggregateFunctions/BFloat16Histogram.h>
+
+namespace DB
+{
+namespace ErrorCodes
+{
+    extern const int NOT_IMPLEMENTED;
+}
+
+template <typename Value>
+struct QuantileBFloat16Histogram
```

Comment:
It's pointless to have this small wrapper class. We can implement BFloat16Histogram directly here.